Statistics/Validation/TriangularValidator.cs

```diff
@@ -22,28 +22,28 @@ public IEnumerable<IMessage> ReportErrors(Triangular obj)
         {
             List<IMessage> msgs = new List<IMessage>();
             if (obj.IsNull()) throw new ArgumentNullException(nameof(obj), "The triangular distribution cannot be validated because it is null.");
-            if (obj.Range.Min == obj.Range.Max) msgs.Add(IMessageFactory.Factory(IMessageLevels.Message, $"The {obj.Print(true)} minimum and maximum values are equal. This is allowed but reduces the distribution to a constant."));
+            if (obj.Min == obj.Max) msgs.Add(IMessageFactory.Factory(IMessageLevels.Message, $"The {obj.Print(true)} minimum and maximum values are equal. This is allowed but reduces the distribution to a constant."));
```

Comment:
if min = max, we could be in a situation where we divide by zero? do we need logic that converts the triangular distribution into a deterministic distribution?

Reply:
yeah right now this just ends up throwing an exception i think.